src/Symfony/Component/Console/Command/Command.php

```diff
@@ -548,7 +545,7 @@ public function getProcessedHelp()
      *
      * @throws InvalidArgumentException When an alias is invalid
      */
-    public function setAliases($aliases)
+    public function setAliases(?array $aliases)
```

Comment:
Not sure about this change because of the later `if (!\is_array($aliases) && !$aliases instanceof \Traversable) {` check 🤔

Reply:
indeed, should be `iterable`

Reply:
(and the type check can then be removed)

Reply:
the test has been removed too